### ISSUE TITLE
Facebook logout doesn't work

### ISSUE THREAD

**Issue by MattSHallatt** [ShareKit/ShareKit#293]
So working on implementing ShareKit in my latest iOS app and Twitter and email works fine at the moment. But facebook is where I'm struggling. Got the log in working fine and the posting too, the issue comes when I try to get the current facebook user logged out. The app I'm creating will be used by many people and so people will need to be able to log in and out of facebook from within the app but none of the available methods seem to be able to do so unless you go out and quit the tab in safari. I have no more luck when using the facebook application instead, same problem.

So to summarise: The issue is that I can't get the user currently logged in to facebook to log out from within my app and I need to be able to.

Does anybody know if there's just something I'm forgetting to do or can anybody think of ways I could attempt to go about implementing this myself?

Thanks,
Matt :D
